src/scheduler/workers/unpublish/dayBefore/sendDayBeforeProviderConfirmation.ts

```diff
@@ -0,0 +1,73 @@
+import { ListItemJsonData } from "server/models/listItem/providers/deserialisers/types";
+import { schedulerLogger } from "scheduler/logger";
+import { NotifyClient, RequestError } from "notifications-node-client";
+import { NOTIFY } from "server/config";
+import { addUnpublishProviderReminderEvent } from "./addUnpublishProviderReminderEvent";
+import { providerReminderPersonalisation } from "./dayBeforeReminderPersonalisation";
+import { ListItem } from "@prisma/client";
+import { Meta } from "scheduler/workers/types";
+
+const template = NOTIFY.templates.unpublishNotice.providerOneDay;
+
+const notifyClient = new NotifyClient(NOTIFY.apiKey);
+
+export async function sendDayBeforeProviderConfirmation(listItem: ListItem, meta: Meta) {
```

Comment:
these emails are confirming that the list items have been unpublished rather than reminding them review their record.  I can change just to keep consistency if you'd rather but I think it's justified as it is